funnel/views/event.py

```diff
@@ -28,7 +28,7 @@ def admin(profile, space):
     csrf_form = forms.Form()
     if csrf_form.validate_on_submit():
         for ticket_client in space.ticket_clients:
-            if ticket_client and ticket_client.name == u'explara':
+            if ticket_client and ticket_client.name.lower() in [u'explara', u'boxoffice']:
```

Comment:
Don't do `lower()` here. It should be validated when saving to the db. Add a migration if you have bad data.